packages/eslint-plugin-shopify/lib/config/rules/lodash.js

```diff
@@ -0,0 +1,84 @@
+// see https://github.com/wix/eslint-plugin-lodash
+
+module.exports = {
+  // Possible errors
+
+  // Use or avoid thisArg for Lodash method callbacks, depending on major version.
+  'callback-binding': 'warn',
+  // Use value returned from collection methods properly.
+  'collection-method-value': 'warn',
+  // Always return a value in iteratees of Lodash collection methods that aren't forEach.
+  'collection-return': 'error',
+  // Do not use .value() on chains that have already ended (e.g. with max() or reduce()) (fixable)
+  'no-double-unwrap': 'error',
+  // Do not use superfluous arguments on Lodash methods with a specified arity.
+  'no-extra-args': 'error',
+  // Prevent chaining without evaluation via value() or non-chainable methods like max().
+  'unwrap': 'error',
+
+  // Stylistic issues
+
+  // Enforce a specific chain style: explicit, implicit, or explicit only when necessary.
+  'chain-style': ['warn', 'explicit'],
+  // Prefer identity shorthand syntax
+  'identity-shorthand': ['warn', 'never'],
+  // Prefer matches property shorthand syntax
+  'matches-prop-shorthand': ['warn', 'never'],
+  // Prefer matches shorthand syntax
+  'matches-shorthand': ['warn', 'never'],
+  // Do not use .commit() on chains that should end with .value()
+  'no-commit': 'warn',
+  // Prevent chaining syntax for single method, e.g. _(x).map().value()
+  'no-single-chain': 'warn',
```

Comment:
I think you are. When you begin a chain with `_(x)` it returns a special lodash object that allows chaining to happen. Certain operations are "final", like `join`, `first`, `last` (full list [here](https://lodash.com/docs#_)), but if you don't finish with any of them you'll just end up with a lodash object instead of your array/string/object/whatever. That's when you'd call `.value()`, so you get the actual result back. This ensures you don't needlessly chain